deploy/dep.yaml

```diff
@@ -33,6 +33,12 @@ rules:
   - apiGroups: [""]
     resources: [nodes]
     verbs: [get, watch, list, update]
+  - apiGroups: [""]
+    resources: [secrets]
+    verbs: [get, list, watch]
+  - apiGroups: [""]
+    resources: [configmaps]
+    verbs: [get, list]
```

Comment:
Do we need to make these changes to the [initcontainer role](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/blob/master/assets/rbac/initcontainer_role.yaml) that the operator creates?

Reply:
That would allow a label updater to real all Secrets in a cluster, that's now what we want. The labeller reads only Secrets in its own namespace and that's allowed, at least the labeller [does not complain](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_ibm-vpc-node-label-updater/18/pull-ci-openshift-ibm-vpc-node-label-updater-master-e2e-ibmcloud-csi/1622545665127092224/artifacts/e2e-ibmcloud-csi/gather-extra/artifacts/pods/openshift-cluster-csi-drivers_ibm-vpc-block-csi-node-m4r94_vpc-node-label-updater.log) in our e2e tests.

Good catch, btw.